cmd/status-backend/requests/AcceptContactRequest.http

```diff
@@ -0,0 +1,13 @@
+### Acceot Contact Request
+# @name AceptContactRequest
+POST {{address}}/statusgo/CallRPC
+
+{
+    "jsonrpc": "2.0",
+    "method": "wakuext_acceptContactRequest",
+    "params": [
+        {
+            "id": "0x0495811d8ac62e240e9ae7f05fa097938032066ff65764d97dc93a73d83262d884a75696520543899b047005f6fb91af42dcb04f635be760313cc4d26fa27f1ff7"
```

Comment:
Shouldn't it be parameterized as well?

Reply:
It should, but I was unsure of how we're going to use it, so decided to leave as is for now.
Like first see where the people go and then make concrete sidewalks on top of footpaths